sources/api/migration/migrations/v1.1.3/cpu-manager-policy/src/main.rs

```diff
@@ -0,0 +1,65 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::{error, migrate, Migration, MigrationData, Result};
+use snafu::ResultExt;
+use std::fs;
+use std::path::Path;
+use std::process;
+
+const SETTING: &str = "settings.kubernetes.cpu-manager-policy";
+const CPU_MANAGER_POLICY_CHECKPOINT: &str = "var/lib/kubelet/cpu_manager_state";
+/// We remove cpu manager policy checkpoint value if checkpoint value is differenet from upgrade version
+/// or downgrade version cpu-manager-policy setting value.
+pub struct CpuManagerPolicyCleaner;
+
+impl Migration for CpuManagerPolicyCleaner {
+    fn forward(&mut self, input: MigrationData) -> Result<MigrationData> {
+        // removing existing cpu_manager_policy_state file
+        if Path::new(CPU_MANAGER_POLICY_CHECKPOINT).exists() {
+            println!(
+                "Deleting existing cpu manager policy checkpoint: '{}'",
+                CPU_MANAGER_POLICY_CHECKPOINT
+            );
+            fs::remove_file(CPU_MANAGER_POLICY_CHECKPOINT).context(error::RemoveFile {
+                path: CPU_MANAGER_POLICY_CHECKPOINT,
+            })?;
```

Comment:
I don't think the program should exit with error if the file does not exist. I think the right way to do this is to check the std::io::ErrorKind for NotFound and maybe println! if the error is NotFound, otherwise error out. (Same for the `backward` function below as well)

https://doc.rust-lang.org/std/fs/fn.remove_file.html
https://doc.rust-lang.org/std/io/enum.ErrorKind.html

Reply:
Sure. I'll update that! thanks